### PR TITLE
Shilpa|BAH-632|Made httpd 2.4 service running

### PR DESCRIPTION
**Pull request by shilpa-iplit**
https://bahmni.atlassian.net/browse/BAH-632

### CODE REVIEW

File: bahmni-web/resources/httpd.conf [hunk 1]
```diff
@@ -841,201 +336,18 @@ AddOutputFilter INCLUDES .shtml
...
-	RewriteEngine On
-  RewriteCond %{HTTPS} off
-			RewriteCond %{REMOTE_HOST} !(127.0.0.1)
-						RewriteCond %{REQUEST_URI} !(^/jasperserver)
-
-
-      RewriteRule (.*) https://%{HTTP_HOST}%{REQUEST_URI}
-
-	        ProxyPass /jasperserver http://localhost:8080/jasperserver
-	        ProxyPassReverse /jasperserver http://localhost:8080/jasperserver
```
Comment (SwathiVarkala) [inline bahmni-web/resources/httpd.conf]: Is this taken care somewhere else?

File: bahmni-web/resources/ssl.conf [hunk 2]
```diff
@@ -287,26 +287,25 @@ RewriteRule (.*) http://localhost:8069$1 [P]
...
-	            WSGIDaemonProcess client_side_logging user=apache group=apache threads=1
+            Redirect permanent /home /bahmni/home/
+                WSGIDaemonProcess client_side_logging user=apache group=apache threads=1
             WSGIScriptAlias /log /var/www/client_side_logging/client_side_logging.wsgi
 
             <Directory /var/www/client_side_logging>
             WSGIProcessGroup client_side_logging
             WSGIApplicationGroup %{GLOBAL}
-            Order deny,allow
-            Allow from all
```
Comment (SwathiVarkala) [inline bahmni-web/resources/ssl.conf]: Can you explain what this change is doing?

File: bahmni-web/resources/httpd.conf [hunk 3]
```diff
@@ -841,201 +336,18 @@ AddOutputFilter INCLUDES .shtml
...
-AddOutputFilterByType DEFLATE text/html
-AddOutputFilterByType DEFLATE text/plain
-AddOutputFilterByType DEFLATE text/xml
-AddOutputFilterByType DEFLATE text/css
-AddOutputFilterByType DEFLATE application/xml
-AddOutputFilterByType DEFLATE application/xhtml+xml
-AddOutputFilterByType DEFLATE application/javascript
-AddOutputFilterByType DEFLATE application/x-javascript
-AddOutputFilterByType DEFLATE application/json
-AddOutputFilterByType DEFLATE text/javascript
```
Comment (SwathiVarkala) [inline bahmni-web/resources/httpd.conf]: Are these not required ?

File: bahmni-web/resources/httpd.conf [hunk 4]
```diff
@@ -841,201 +336,18 @@ AddOutputFilter INCLUDES .shtml
...
-    <Directory "/var/www/error">
-        AllowOverride None
-        Options IncludesNoExec
-        AddOutputFilter Includes html
-        AddHandler type-map var
-        Order allow,deny
-        Allow from all
-        LanguagePriority en es de fr
-        ForceLanguagePriority Prefer Fallback
-    </Directory>
```
Comment (SwathiVarkala) [inline bahmni-web/resources/httpd.conf]: is this config not required any more?

File: bahmni-web/resources/httpd.conf [hunk 5]
```diff
@@ -493,341 +188,141 @@ ErrorLog "|/usr/sbin/rotatelogs /var/log/httpd/error_log 10M"
...
-AddHandler type-map var
-
-#
-# Filters allow you to process content before it is sent to the client.
-#
-# To parse .shtml files for server-side includes (SSI):
-# (You will also need to add "Includes" to the "Options" directive.)
-#
-AddType text/html .shtml
-AddOutputFilter INCLUDES .shtml
```
Comment (SwathiVarkala) [inline bahmni-web/resources/httpd.conf]: Please check

**Comment by SwathiVarkala** [comment]
@shilpa-iplit 
Can you please give details on how you have approached this?
Have you compared the old httpd.conf and ensured all the bahmni specific changes are put in?

**Comment by shilpa-iplit** [comment]
@SwathiVarkala Have copied the newly generated httpd 2.4 conf but missed making Bahmni specific changes in it. So will make changes as per your suggestions.

**Pull request merged**